src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -0,0 +1,295 @@
+
+        ```sql
+        ... where traceId = '030a573f0df02c57'
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Attribute names with spaces
+      </td>
+
+      <td>
+        Use backticks ``` `` ``` to quote a [custom attribute](/docs/insights/new-relic-insights/decorating-events/insights-custom-attributes) name that has a space in it. For example:
+
+        ```sql
+        ... FACET `Logged-in user`
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Data type coercion
+      </td>
+
+      <td>
+        We don't support data type "coercion." For more information, see [Data type conversion](/docs/insights/nrql-new-relic-query-language/nrql-reference/nrql-syntax-components-functions#type-conversion).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Use of math functions
+      </td>
+
+      <td>
+        [Basic and advanced math functions](/docs/insights/nrql-new-relic-query-language/using-nrql/nrql-math-using-select) are supported in the [`SELECT`](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#state-select) statement.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Subqueries
+      </td>
+
+      <td>
+        NRQL supports [subqueries](/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `JOIN` clauses
+      </td>
+
+      <td>
+        NRQL does not have the equivalent of the SQL `JOIN` clause, but you can [simulate a `JOIN` with custom attributes](/docs/insights/new-relic-insights/using-new-relic-query-language/simulate-sql-join-functions-insights).
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+These are the basic rules for NRQL, and we have more information on [NRQL syntax and functions](/docs/query-data/nrql-new-relic-query-language/getting-started/nrql-syntax-components-functions) to help you build your queries.
+
+## Data exploration [#explore-data]
+
+One of the best ways to learn how to use NRQL is to access one of the [query tools](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#where) and use it to explore your data. Here's an example of exploring your data using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) and the suggested entries from the interface. 
+
+The query begins with `FROM` followed by space. The interface suggests available types of data, and `Transaction` is chosen from that list.
+
+<img
+  title="nrql_query01.png"
+  alt="A screenshot of a NRQL query beginning with From"
+  src={queriesnrqlEventDefinitionsQueryBuilder}
+/>
+
+Next, an [attribute](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) is chosen using `SELECT`, and the query looks as follows:
+
+```sql
+FROM Transaction SELECT
+```
+
+Pressing space again causes the interface to suggest available attributes. `appId` is chosen.
```

Comment:
What if you said, "In the example below, `appId` is chosen?

Reply:
Agreed and changed